src/legacy/core_plugins/kibana/public/dashboard/panel/dashboard_panel.tsx

```diff
@@ -71,7 +71,7 @@ class DashboardPanelUi extends React.Component<DashboardPanelUiProps, State> {
         ? null
         : props.intl.formatMessage({
             id: 'kbn.dashboard.panel.noEmbeddableFactoryErrorMessage',
-            defaultMessage: 'No factory found for embeddable',
+            defaultMessage: 'The feature to render this panel is missing.',
```

Comment:
A bit less technical error message, and also under regular circumstances this should rarely ever been visible to a user.